src/main/kotlin/no/nav/hjelpemidler/delbestilling/infrastructure/email/Email.kt

```diff
@@ -9,10 +9,11 @@ class Email(
     val client: GraphClientInterface,
 ) {
 
-    suspend fun sendSimpleMessage(
+    suspend fun send(
         recipentEmail: String,
         subject: String,
         bodyText: String,
+        contentType: ContentType,
```

Comment:
Ta inn contentType for å kunne velge om det skal sendes som plaintext eller HTML